plugins/uast-kotlin-base/src/org/jetbrains/uast/kotlin/internal/baseUastMultiResolve.kt

```diff
@@ -0,0 +1,16 @@
+/*
+ * Copyright 2010-2021 JetBrains s.r.o. and Kotlin Programming Language contributors.
+ * Use of this source code is governed by the Apache 2.0 license that can be found in the license/LICENSE.txt file.
+ */
+
+package org.jetbrains.uast.kotlin.internal
+
+import com.intellij.psi.PsiSubstitutor
+import com.intellij.psi.ResolveResult
+import com.intellij.psi.infos.CandidateInfo
+import org.jetbrains.uast.UMultiResolvable
+import org.jetbrains.uast.UResolvable
+
+interface DelegatedMultiResolve : UMultiResolvable, UResolvable {
```

Comment:
A strange name for a file with the only class named `DelegatedMultiResolve`

Reply:
Right... I assumed that more definitions around `DelegatedMultiResolve` can be commonized as well, but they could be placed in a separate file anyway. Renamed to the interface name.